lib/mix/tasks/eliver.bump.ex

```diff
@@ -6,36 +6,129 @@ defmodule Mix.Tasks.Eliver.Bump do
     args |> parse_args |> bump
   end
 
-  defp bump(_) do
+  defp bump(args) do
     case check_for_git_problems() do
       {:error, message} ->
         say message, :red
       {:ok} ->
-        {new_version, changelog_entries} = get_changes()
-        if allow_changes?(new_version, changelog_entries) do
-          make_changes(new_version, changelog_entries)
+        do_bump(args)
+    end
+  end
+
+  defp do_bump(args) do
+    case Keyword.get(args, :multi) do
+      true -> bump_multiple()
+      nil -> bump_normal()
+    end
+  end
+
+  defp bump_multiple() do
```

Comment:
It could make sense to try to split this one into smaller functions so that it's a bit more understandable what it does, wdyt ?